app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import { useSelector } from 'react-redux';
+
+const Greeting = () => {
+  const { message, isLoading, isError, errorMsg } = useSelector((store) => store.message);
+
+  if (isLoading) {
+    return (<h1>Loading Greeting</h1>);
+  }
+
+  if (isError) {
+    return (<h1>{ errorMsg }</h1>);
+  }
+
+  return (
+    <h1>{ message }</h1>
+  )
+}
+export default Greeting
```

Comment:
- [ ] Nothing is displayed on the page when I visit the Greeting page. Only a Loading Greeting text shows up and disappears, but the greeting message itself is not shown at all. Only the console log from the message slice shows up in the console tab.

  ![image](https://github.com/Wineshuga/hello-rails-react/assets/36542986/f7aa6caa-b2ca-4316-9f54-1ba787ad0bee)